draft-pda-core.md

```diff
@@ -102,41 +128,44 @@ document.
 
 # Overview {#overview}
 
-The protocol is executed by a large set of clients and a small set of
-servers.  We call the servers the *aggregators*. Each client's input to
-the protocol is a set of measurements (e.g., counts of some user behavior).
-Given the input set of measurements x[1], ..., x[n] held by n users, the goal is to
-compute y = F(x[1], ..., x[n]) for some aggregation function F, while
-revealing nothing else about the measurements.
-
-
-## Secret sharing
-
-Prio achieves this goal using additive secret sharing. Rather than send its
-input in the clear, each client "splits" its measurements into a sequence of "shares"
-and sends a share to each of the aggregators. This secret-sharing procedure has
-two important properties: first, it is impossible to deduce the measurement given only
-a proper subset of the shares; and second, it allows the aggregators to compute
-the final output by first adding up their measurements shares locally, then combining
-the results to obtain the final output.
-
-Consider an illustrative example. Suppose there are three clients and
-two aggregators. Each client i holds a single measurement in the form of
-a positive integer x[i], and our goal is to compute the sum of the
-measurements of all clients. In this case, the protocol input is a single measurement consisting of
-a single positive integer; no additional encoding is done. Given this input, the first client splits its
-measurement x[1] with additive secret-sharing into a pair of integers x[1,1] and x[1,2] for which
-x[1] = x[1,1] + x[1,2] modulo a prime p. (For convenience, we will omit the
-the "mod p" in the rest of this section.) It then uploads x[1,1] to
-one sever x[1,2] to the other. The second client splits its measurement x[2]
-into x[2,1] and x[2,2], uploads them to the servers, and so on.
-
-Now the first aggregator is in possession of shares x[1,1], x[2,1], and
-x[3,1], and the second aggregator is in possession of shares x[1,2],
-x[2,2], and x[3,2]. Each aggregator computes the sum of its  shares. Let
-A[1] denote the first aggregator's share of the sum and let A[2] denote the
+The protocol is executed by a large set of clients and a small set of servers.
```

Comment:
Small = 2?

Reply:
Small = O(1). Remember that the plan is to execute the protocol among multiple sets of two servers.

Reply:
Yes, I'm aware. My point was that the core protocol runs with s=2 servers.

Reply:
Ah, gotcha. I still think "small set of servers" is more accurate than "two servers" here, since this section is meant to describe the general of case of one leader and multiple helpers.

Reply:
Sounds good!